release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -1765,3 +1765,52 @@ link:https://access.redhat.com/solutions/4627461[{product-title} 4.2.9 container
 
 To upgrade an existing {product-title} 4.2 cluster to this latest release, see
 xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
+
+[[ocp-4-2-10]]
+=== RHBA-2019:4092 - {product-title} 4.2.10 Bug Fix Update
+
+Issued: 2019-12-10
+
+{product-title} release 4.2.10 is now available. The list of packages included in
+the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2019:3952[RHBA-2019:4092] advisory.
+The container images and bug fixes included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2019:3953[RHBA-2019:4093] advisory.
+
+Space precluded documenting all of the container images for this release in the
```

Comment:
KB article link from @sheriff-rh to be added here, which will include the container images.

Reply:
@ahardin-rh https://access.redhat.com/solutions/4643781